src/System.IO.Packaging/src/System/IO/Packaging/InternalRelationshipCollection.cs

```diff
@@ -27,6 +27,12 @@ namespace System.IO.Packaging
     /// </summary>
     internal class InternalRelationshipCollection : IEnumerable<PackageRelationship>
     {
+        // Mono will parse a URI starting with '/' as an absolute URI, while .NET Core and
+        // .NET Framework will parse this as relative. This will break internal relationships
+        // in packaging. For more information, see
+        // http://www.mono-project.com/docs/faq/known-issues/urikind-relativeorabsolute/
+        private static readonly UriKind DotNetRelativeOrAbsolute = Type.GetType ("Mono.Runtime") == null ? UriKind.RelativeOrAbsolute : (UriKind)300;
```

Comment:
This is unfortunate. 😦

Reply:
cc: @tmds, @akoeplinger

Reply:
I assume the check is done at runtime because System.IO.Packaging.dll is an OOB library that can run on Mono?

We don't yet use the code for the System.IO.Packaging namespace from corefx in Mono so if the goal is just to do that in the future we could go with an #ifdef.

Reply:
@akoeplinger We use this package in [DocumentFormat.OpenXml](https://github.com/OfficeDev/Open-XML-SDK) and have a .NET Standard build. When someone on Xamarin pulls in the library, they end up with the .NET Standard build, which uses this implementation. So, they end up using this implementation instead of the one in Mono

Reply:
@twsouthwick ok, makes sense

Reply:
@weshaggard, @terrajobst, what is the recommended way for library code to detect it's on Mono?

Reply:
I don't think we have a recommended way. Perhaps @marek-safar  or @akoeplinger might have a suggestion on how to detect it.

Reply:
FWIW the highlighted code is what Mono recommends at http://www.mono-project.com/docs/faq/known-issues/urikind-relativeorabsolute/, but if there's a better way, I'll change it

Reply:
Ok, we can go with this for now.